### PR TITLE
Convert to auto-property

### PR DESCRIPTION
**Pull request by gpetrou**

### CODE REVIEW

File: Src/Microsoft.Dynamic/Runtime/CompilerContext.cs [hunk 1]
```diff
@@ -22,46 +22,13 @@ namespace Microsoft.Scripting.Runtime {
     /// from this class to provide additional contextual information.
     /// </summary>
     public sealed class CompilerContext {
+        public SourceUnit SourceUnit { get; }
 
-        /// <summary>
-        /// Source unit currently being compiled in the CompilerContext
```
Comment (slozier) [inline Src/Microsoft.Dynamic/Runtime/CompilerContext.cs]: Can we move these comments to the properties?

File: Src/Microsoft.Dynamic/zlib.net/ZStream.cs [hunk 2]
```diff
@@ -121,40 +121,22 @@ public sealed class ZStream
...
-        /// </summary>
-        private BlockType data_type;
-		
-        /// <summary>
-        /// A checksum computed with Adler algorithm
-        /// </summary>
-        private long _adler;
 
-        #endregion
+	    #endregion
```
Comment (slozier) [inline Src/Microsoft.Dynamic/zlib.net/ZStream.cs]: Is there an indentation issue here? (Same thing for the summaries below).

File: Src/Microsoft.Dynamic/Runtime/CompilerContext.cs (hunk 1 above)
Comment (gpetrou) [inline Src/Microsoft.Dynamic/Runtime/CompilerContext.cs]: Sure. Should we follow StyleCop rules for such comments? "Gets or sets etc..." http://stylecop.soyuz5.com/SA1623.html

Comment (slozier) [inline Src/Microsoft.Dynamic/Runtime/CompilerContext.cs]: We probably should, but I expect most of the codebase doesn't so I'm not too worried about it.

File: Src/Microsoft.Dynamic/zlib.net/ZStream.cs (hunk 2 above)
Comment (gpetrou) [inline Src/Microsoft.Dynamic/zlib.net/ZStream.cs]: Apparently there are some tabs from previous code. I will replace some of them with whitespace.

**Comment by gpetrou** [comment]
Can we merge this?

**Pull request merged**